doc/articles/features/windows-devices-sensors.md

```diff
@@ -9,22 +9,31 @@ The following APIs are implemented on Android, iOS and WASM:
 * `ReportInterval`
 
 ### Implementation notes
+
 While iOS features a built-in shake gesture recognition, this is not available on Android and WASM. For these platforms we use a commonly used implementation which tries to approximate the shake motion to detect it. Both Android and WASM use the same implementation. In case this is not sufficient for your use case, you can implement Shake detection above the `ReadingChanged` event.
 
 `ReportInterval` property on WASM is not supported directly and we use an approximation in the form of raising the `ReadingChanged` event only when enough time has passed since the last report. The event is actually raised a bit more often to make sure the gap caused by the filter is not too large, but this is in-line with the behavior of UWP Accelerometer.
 
 On Android, when both `ReadingChanged` and `Shaken` events are attached and the user sets the `ReportInterval` to a high value, the `ReadingChanged` event may be raised more often than requested. This is because for multiple subscribers to the same sensor the system may raise the sensor events with the frequency of the one with lower requested report delay. This is, however, again in line with the behavior of UWP Accelerometer and you can filter the events as necessary for your use case.
 
-
 ## `Barometer`
 
 ### Limitations
 
-**Android**
-- Only `ReadingChanged` event is supported, system does not support retrieval of a single reading
+#### Android
+
+* Only `ReadingChanged` event is supported, system does not support retrieval of a single reading
+
+#### iOS
+
+* Only `ReadingChanged` event is supported, system does not support retrieval of a single reading
 
-**iOS**
-- Only `ReadingChanged` event is supported, system does not support retrieval of a single reading
+#### WASM
+
+* No barometer API is currently available in JS
+
+## `Magnetometer`
+
+### Limitations
```

Comment:
mention here that accuracy isn't reported on iOS

Reply:
Done :-)